tests/security/pam/pam_login.pm

```diff
@@ -0,0 +1,71 @@
+# Copyright (C) 2020 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+#
+# Summary: PAM tests for login, user login should fail without authentication
+# Maintainer: rfan1 <richard.fan@suse.com>
+# Tags: poo#70345, tc#1767577
+
+use base 'opensusebasetest';
+use strict;
+use warnings;
+use testapi;
+
+sub run {
+    my $self = shift;
+    $self->select_serial_terminal;
+
+    # Define the user and password, which are already configured in previous milestone
+    my $user   = 'suse';
```

Comment:
just a question, if you'd like to add another test user which is not in deniedusers and can log in correctly without a problem.

Reply:
Good suggestion, Will do